<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelEdit</class>
 <widget class="QWidget" name="ModelEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(244, 244, 244);
}



</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>271</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
	background-color: rgb(116, 126, 146);
	border-radius: 12px;
}
QLineEdit
{	
	color: rgb(255, 255, 255);
	border-radius: 4px;
	border: 1px solid rgb(255,250,250);
	background-color: rgb(116, 126, 146);
}
QLabel
{
	font-family:&quot;Source Sans Pro&quot;;
	font-size: 12px;
    font-weight: regular;
	color:#FFF;
}

QPushButton,QToolButton
{
   /* color: #424344;*/
    font-family: &quot;Source Sans Pro&quot;;
    font-size: 11px;
    font-weight: regular;
 	border: 1px solid yellow;
 	border-radius: 4px;
    padding: 0px 15px;   
    max-height: 100px;
    min-height: 25px;
	min-weight:52px;

 	/*background-color: #2288C1;*/
	background-color: rgb(155,155,155);
    border-color: rgb(155,155,155);
    color: #fff;
}

QToolButton:pressed, QPushButton:pressed, QPushButton[btn-secondary=true]:pressed, QPushButton[btn-icon=true]:pressed
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #E0E1E3, stop:.25 #F6F7F9);
}

QPushButton:disabled
{
    background-color: #EAEBED;
    border-color: #E1E2E4;
    color: #C0C1C3;
}

QPushButton:pressed,QToolButton:pressed
{
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #0072a9, stop:.25 #2288C1);
}
QPushButton:hover,QToolButton:hover
{
    background-color: #6C22E2;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>244</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="AddModelButton">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Add  new model(s) </string>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="DuplicateModeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Duplicate the selected model</string>
       </property>
       <property name="text">
        <string>复制</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="RemoveModelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Delete the seleted object</string>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="ModelMoveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Change to top view,move the object</string>
       </property>
       <property name="text">
        <string>移动</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ModelRotButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>旋转</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="RestRotButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Reset the rotation .</string>
       </property>
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ColorButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="text">
        <string>颜色</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="ModelSelectedButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Change to select mode</string>
       </property>
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QToolButton" name="ModelScaleButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Scaled the selected object</string>
       </property>
       <property name="text">
        <string>缩放</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_scale">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{	
	
}</string>
       </property>
       <property name="text">
        <string>1.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="SupZ">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>102</height>
        </size>
       </property>
       <property name="text">
        <string>升降</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="SupZ_linedit">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>126</y>
      <width>246</width>
      <height>87</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <property name="horizontalSpacing">
      <number>2</number>
     </property>
     <property name="verticalSpacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="ModelExportButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Export the model to STL file</string>
       </property>
       <property name="text">
        <string>保存模型</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QListWidget" name="ModelList">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>147</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	color: rgb(255, 255, 255);	
	background-color: rgb(115, 115, 115);
    font-family: &quot;Source Sans Pro&quot;;
    font-size: 14px;
    font-weight: regular;
}</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="ModelSnapButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>81</width>
         <height>102</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Sans Pro</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Drop the object to the floor</string>
       </property>
       <property name="text">
        <string>放置平台</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>219</y>
      <width>251</width>
      <height>36</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_rot">
       <property name="maximumSize">
        <size>
         <width>58</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>旋转角度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_rotx">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_roty">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_rotz">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>52</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>260</y>
      <width>251</width>
      <height>39</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>Size:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sizex">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sizey">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sizez">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
